examples/ethereum/config/config.go

```diff
@@ -249,15 +249,15 @@ func LoadConfiguration() (*configuration.Configuration, error) {
 	//}
 	config.Port = 8080
 
-	payload := []configuration.Token{}
+	// payload := []configuration.Token{}
 	config.RosettaCfg = configuration.RosettaConfig{
 		SupportRewardTx: true,
 		TraceType:       configuration.GethNativeTrace,
 		Currency: &RosettaTypes.Currency{
 			Symbol:   "ETH",
 			Decimals: 18,
 		},
-		TokenWhiteList: payload,
+		// TokenWhiteList: payload,
```

Comment:
we need whitelist feature, otherwise rosetta-ethereum wont work

Reply:
ah i'll uncomment this out